discord/emoji.py

```diff
@@ -141,8 +141,10 @@ class Emoji(Hashable):
         If this emoji is managed by a Twitch integration.
     guild_id: :class:`int`
         The guild ID the emoji belongs to.
+    user :class:`User`
+        The user that created the emoji.
```

Comment:
Needs a wording update that mentions that it's not always available and the permission required for it to be received.